### PR TITLE
Update ksproperty-cameracontrol-extended-backgroundsegmentation.md to use match with SDK

### PR DESCRIPTION
**Pull request by Sudhaushu**
renamed KSCAMERA_EXTENDEDPROP_BACKGROUNDSEGMENTATION_PORTRAITMODE to KSCAMERA_EXTENDEDPROP_BACKGROUNDSEGMENTATION_SHALLOWFOCUS as per the SDK.

### CODE REVIEW

File: windows-driver-docs-pr/stream/ksproperty-cameracontrol-extended-backgroundsegmentation.md [hunk 1]
```diff
@@ -26,14 +26,14 @@ Examples of setting KSPROPERTY controls can be found in the [AVStream Camera Sam
...
 
 The following table shows example images of how this works:
 
 | Mode | Example image |
 |--|--|
-| **Portrait & Blur Off** | ![background segmentation portrait blur off.](images/backgroundsegmentation-portrait-blur-off.png) |
-| **Portrait On** | ![background segmentation portrait on.](images/backgroundsegmentation-portrait-on.png) |
+| **Shallow Focus & Blur Off** | ![background segmentation portrait blur off.](images/backgroundsegmentation-portrait-blur-off.png) |
+| **Shallow Focus On** | ![background segmentation portrait on.](images/backgroundsegmentation-portrait-on.png) |
```
Comment (kasumman) [inline windows-driver-docs-pr/stream/ksproperty-cameracontrol-extended-backgroundsegmentation.md:36]: Should it be here with the Blur flag to be clear how it is used.
```suggestion
| **Shallow Focus On & Blur** | ![background segmentation portrait on.](images/backgroundsegmentation-portrait-on.png) |
```

Comment (Sudhaushu) [reply windows-driver-docs-pr/stream/ksproperty-cameracontrol-extended-backgroundsegmentation.md]: made a few changes here. please review.

**Review by strongstrongyiwen — COMMENTED** [review]
One more portrait mode: "The following flags can be placed in the KSCAMERA_EXTENDEDPROP_HEADER.Flags field to control portrait mode."

**Comment by Sudhaushu** [comment]
Thanks!

---
In reply to: [994279646](https://github.com/MicrosoftDocs/windows-driver-docs/pull/3157#issuecomment-994279646) [](http://example.com/codeflow?ancestors=994279646,994279646)

**Review by LPBourret — APPROVED** [review]

**Review by barrygolden — APPROVED** [review]
@Sudhaushu, looks good. Let me know when the required reviewers have sighed off and the PR is ready to merge.

**Comment by Sudhaushu** [comment]
@barrygolden : This PR is ready to merge. Thanks!

---
In reply to: [1000357048](https://github.com/MicrosoftDocs/windows-driver-docs/pull/3157#issuecomment-1000357048) [](http://example.com/codeflow?ancestors=1000357048,1000357048)

**Pull request merged**